src/components/ResearchGroupCard/style.scss

```diff
@@ -1,23 +1,120 @@
-.App {
-  text-align: center;
+$card-header-font: 24px;
+$card-header-height: 64px;
+
+.card{
+    text-align: center;
+    font-family: 'Futura', sans-serif;
+    background-color: #fafafa;
+    border-style: solid;
+    border-color: #cecece;
+    margin: 10px;
+    height: 450px;
+    min-width: 200px;
+    overflow-wrap: break-word;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.card_professor{
+    text-align: left;
+    color: #5dadf7;
+    margin-left: 20px;
+    margin-right: 20px;
+    font-size: 16px;
+    height: 5%;
+    margin-top: 20px;
+    overflow-y: hidden;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
+.card_research-group{
+    text-align: left;
+    font-weight: bold;
+    margin-left: 20px;
+    margin-right: 20px;
+    margin-top: 2px;
+    margin-bottom: 15px;
+    font-size: $card-header-font;
+   height: $card-header-height;
+   text-overflow: ellipsis;
+   overflow-y: hidden;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: #61dafb;
+.card_content{
+    text-align: left;
+    font-weight: 100;
+    margin-left: 20px;
+    margin-right: 20px;
+    margin-top: 2px;
+    font-size: 14px;
+    height: 50%;
+    color: #888888;
+    text-overflow: ellipsis;
+}
+
+.card_courses{
+  text-align: left;
+  margin-left: 20px;
+  margin-bottom: 20px;
+  font-size: 14px;
+  font-weight: 100;
+    color: #888888;
+}
+.card_courses p{
+  margin:0px;
+    color: #5dadf7;
+    font-size: 14px;
 }
 
-.App-link {
-  color: #61dafb;
+.card hr {
+    border: none;
+    height: 3px;
+    background-color: #cecece; 
 }
+
+p { 
```

Comment:
if you're going to do a reset like this, create a `global.scss` in `src/styles` and import it before everything else.

Reply:
Removed it, as it is already present in normalize.css!